File: frontend/src/configs/urls/urls.js
const BASE_URL = '/api/v1'


const urls = {
    USERS : '/users',
    SERVICES: '/services',
    ORDERS: '/orders',
    AUTH: {
        auth: '/auth',
        refresh: '/auth/refresh',
        token_check: '/auth/token_check',
        request_password_recovery: '/auth/request_password_recovery',
        stripe_token: 'auth/stripe_token',
        activate_request: '/auth/activate_request'
    }
}

export {
    BASE_URL, urls
}

File: backend/apps/auth/urls.py
from rest_framework_simplejwt.views import TokenRefreshView

from django.urls import path

from .views import (
    ActivateUserByTokenView,
    CustomTokenObtainPairView,
    RecoveryPasswordByTokenView,
    RequestOfActivationLinkView,
    RequestRecoveryPasswordView,
    StripeTokenView,
    TokenValidCheck,
)

urlpatterns = [
    path('', CustomTokenObtainPairView.as_view()),
    path('/refresh', TokenRefreshView.as_view()),
    path('/<str:token>/activate', ActivateUserByTokenView.as_view()),
    path('/request_password_recovery', RequestRecoveryPasswordView.as_view()),
    path('/<str:token>/recovery', RecoveryPasswordByTokenView.as_view()),
    path('/token_check', TokenValidCheck.as_view()),
    path('/stripe_token', StripeTokenView.as_view()),
    path('/activate_request', RequestOfActivationLinkView.as_view())
]
